### DISCUSSION TITLE
Add an option to share a bookmark to other users.

### DISCUSSION THREAD

**Discussion by MiSebCo** [Radiokot/photoprism-android-client#167] [Ideas]
The Gallery App is very useful as it gives a less complex access to the photoprism library. I use it e.g. with my family - and especially the older ones like it a lot.

I would like to point users to specific search results of elements of the library in an easy way - e.g. share via messenger app.

Use cases would be "I created a new Album - have a look at it.". "Look at the pictures from day xxx, please put the ones you like into an album", etc.

Idea: Add an option to share a bookmark to other users.

The user receives the shared bookmark e.g. via a messenger app. Gallery App opens and presents the results of the query to the user. Users can then add the bookmark to their local instance if useful for them. The bookmark should not be automatically added to the bookmarks, as the user might not want to have it. Bookmark name should be included in the sharing to be able to later reference to it. Maybe just show the query and when the user pushes the bookmark option have the name of the bookmark already populated, so that the user can also change it.

**Comment by Radiokot** [comment]
Hey Michael, I think it's quite a narrow use-case, as a shared search query only makes sense to those having the gallery connected to the same library. I'll add this to the list of ideas, however it is unlikely to be added soon.

**Reply by MiSebCo** [reply]
Yes - definitely. Maybe the shared bookmark would also need to have a reference to the library as a generic bookmark might not be useful. I want to use it to facilitate more collaborative work on the library.

**Reply by Radiokot** [reply]
Currently, you can create an album with the photos you want to present, and share the album name. Alternatively, you can share a plain search query, but in this case you must only use the text field

**Reply by MiSebCo** [reply]
Thank you. Very good options - highly appreciated. I currently so use the workaround using albums. With this, however, I get more and more "temporary" albums within the ones I really want to have. Copy pasting the query text is to complicated for the users, although I really like the idea. 

For collaboratively sorting the pictures from a specific event a query like "day:24 month:12 year:2024 unsorted:true" would help. Then this query is also dynamic.
